keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -644,6 +655,12 @@ spec:
     port: 80
   sessionAffinity: None
 status:
+  status:
+    conditions:
```

Comment:
```suggestion
  conditions:
```

Reply:
It's still in the status right now but indeed if we remove the status we will have to do that

Reply:
yeah, it's already in the status, line 657. sorry for the confusion.

Reply:
> It's still in the status right now but indeed if we remove the status we will have to do that

I think `conditions` should likely remain nested under status as this is where a user would expect to find it, regardless of any other changes to fields.